frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -238,9 +239,9 @@ async function onSaveDraft(data: any, isAutoSave = false) {
       throw new Error('Failed to retrieve correct draft data');
     }
 
-    if (isAutoSave) {
+    if (isAutoSave && showToast) {
       toast.success('Draft autosaved');
-    } else {
+    } else if (showToast) {
```

Comment:
Optional Style: May be worth considering a boolean evaluation order inversion to improve scope clarity:
```js
if (showToast) {
  if (isAutoSave) toast.success('Draft autosaved');
  else {
    toast.success('Draft saved');
    formUpdated.value = false;
  }
}
```

Potential side effect: Also unsure about whether we should only be running `formUpdated.value = false` when toasts are allowed to be shown - this may be introducing a subtle unintended bug in behavior where the cleanliness of the form isn't being updated when the toast isn't allowed to display.